### PR TITLE
CI cleanup

### PR DESCRIPTION
**Pull request by pitr-ch**

### CODE REVIEW

File: ci.jsonnet [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 # `jsonnet fmt --indent 2 --max-blank-lines 2 --sort-imports --string-style d --comment-style h -i ci.jsonnet`
 
-# Visual Sturio Code is recommended for editing jsonnet files.
-# It has 'jsonnet' and 'jsonnet Formatter'
-# plugins which makes it a functioning IDE for jsonnet. The default configuration
-# of jsonnet Formatter has to be changed to use double quotes and hash comments.
-# Alternartivelly Atom ca be used, it has syntax hightliting but no formatter.
-
+# The Visual Sturio Code is recommended for editing jsonnet files.
+# It has a 'jsonnet' with a output preview. Alternartivelly Atom can be used.
```
Comment (eregon) [inline ci.jsonnet]: Studio
a 'jsonnet' *plugin*
Alternatively

File: ci.jsonnet [hunk 2]
```diff
@@ -6,7 +6,8 @@
 #
 # Formatter is part of `jsonnet` command, it can be installed with
 # `brew install jsonnet` on mac or with http://linuxbrew.sh/ on a Linux machine
-# with the same command.
+# with the same command. Or it can be downloaded from 
+# https://github.com/google/jsonnet/releases and compile.
```
Comment (eregon) [inline ci.jsonnet]: compiled

**Comment by pitr-ch** [comment]
Ah, I have a tool as well, I could have given you the diff.

* It's there to include the printed versions in the log so it can be checked that the versions are as expected. I don't see any difference in where it's printed. It should certainly not be removed.
* That's part of sulong build it makes sure that sulong uses correct versions, if it's before truffleruby build. Before, sulong had to be build later otherwise it would pickup bad versions, which was quite subtle and fragile.
* As mentioned above, the order does not matter any more. I've noticed it as well during the diff review, but i don't think the order makes any difference. (I think the current order is a result of one of the previous iterations of CI cleanup where a build>sulong order could not have been easily done.)
* Yeah it does not matter, I've ignored the change. Otherwise I would have to decompose other and other-extra benchmark parts into more parts to get the original ordering.

I think the changes are minimal and acceptable. I'd like to avoid doing another commit and waiting for CI again.

**Comment by pitr-ch** [comment]
Ah nevermind, github folded the other 2 comments and I've missed them. I need to fix the typos anyway so I include the ordering changes as well.

**Comment by eregon** [comment]
* Then `mx sversions` should be before `mx build` so if the build fails we know which versions were used (currently it's after `mx build`).
* OK.
* I think it's best to keep the order consistent. All other jobs still build sulong after ruby if they use sulong. So if it's an easy change, please do it. It sounds good to reorder them (for all jobs, consistently), but let's not do it in this refactoring PR if possible.
* OK.

**Comment by pitr-ch** [comment]
Don't merge. I am running all daily and weekly builds as well.

**Pull request merged**